addons/im_livechat/demo/im_livechat_channel/im_livechat_support_bot.xml

```diff
@@ -57,5 +69,68 @@
             <field name="sequence">3</field>
             <field name="script_step_id" ref="chatbot_script_troubleshooting_step_browser_demo"/>
         </record>
+        <record id="chatbot_script_module_selection_demo" model="chatbot.script.step">
+            <field name="message">What module are you interested in?</field>
+            <field name="sequence">3</field>
+            <field name="step_type">question_selection</field>
+            <field name="chatbot_script_id" ref="chatbot_script_support_bot_demo"/>
+        </record>
+        <record id="chatbot_script_answer_discuss_demo" model="chatbot.script.answer">
+            <field name="name">Discuss</field>
+            <field name="sequence">1</field>
+            <field name="script_step_id" ref="chatbot_script_module_selection_demo"/>
+        </record>
+        <record id="chatbot_script_answer_livechat_demo" model="chatbot.script.answer">
+            <field name="name">Livechat</field>
+            <field name="sequence">2</field>
+            <field name="script_step_id" ref="chatbot_script_module_selection_demo"/>
+        </record>
+        <record id="chatbot_script_answer_discuss_livechat_demo" model="chatbot.script.answer">
+            <field name="name">Discuss and Livechat</field>
+            <field name="sequence">3</field>
+            <field name="script_step_id" ref="chatbot_script_module_selection_demo"/>
+        </record>
+        <record id="chatbot_script_module_info_discuss_text_step_demo" model="chatbot.script.step">
+            <field name="message">Hmm, let me check if I can find someone who know Discuss well...</field>
+            <field name="sequence">4</field>
+            <field name="step_type">text</field>
+            <field name="chatbot_script_id" ref="chatbot_script_support_bot_demo"/>
+            <field name="triggering_answer_ids" eval="[(4, ref('chatbot_script_answer_discuss_demo'))]"/>
+        </record>
```

Comment:
I'd rather have a generic text step, it's easier than one per answer. Maybe something like "Let me check if I can connect you with the right expert..." or "Let me find the best person to help you..."?